jumpscale/packages/threebot_deployer/bottle/solutions.py

```diff
@@ -0,0 +1,90 @@
+from beaker.middleware import SessionMiddleware
+from bottle import Bottle, request, HTTPResponse
+
+from jumpscale.loader import j
+from jumpscale.packages.auth.bottle.auth import SESSION_OPTS, login_required, get_user_info
+from jumpscale.packages.marketplace.bottle.models import UserEntry
+from jumpscale.core.base import StoredFactory
+
+app = Bottle()
+
+
+@app.route("/api/threebots/list")
+@login_required
+def list_threebots() -> str:
+    solutions = []
+    user_info = j.data.serializers.json.loads(get_user_info())
+    solutions = j.sals.marketplace.solutions.list_solutions(user_info["username"], "threebot")
+    return j.data.serializers.json.dumps({"data": solutions})
+
+
+@app.route("/api/solutions/count")
+@login_required
+def count_solutions():
+    user_info = j.data.serializers.json.loads(get_user_info())
+    res = j.sals.marketplace.solutions.count_solutions(user_info["username"])
```

Comment:
I:m not sure if this api is still being called. But if it is maybe is should count the result of threebot solutions directly instead of counting all solution types?

Reply:
yes it is not used, removed everything related to it from the frontend. Thank you